lib/src/observable_value.dart

```diff
@@ -0,0 +1,115 @@
+// Copyright (c) 2022, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:async';
+
+/// A single variable with a stream emitting the variable's changing values.
+///
+/// Contains a single [value] which can be read and written as any other
+/// variable.
+///
+/// Whenever the value is set, the new value is emitted on the
+/// [updates] stream.
+/// Further, any new listener on the [updates] stream is initially sent the
+/// current [value].
+///
```

Comment:
maybe we should point out that this has a similar behavior as the [BehaviorSubject](https://pub.dev/documentation/rxdart/latest/rx/BehaviorSubject-class.html) in the RxDart / ReactiveX world.
```suggestion
/// This this a behavior similar to the
/// [BehaviorSubject](https://pub.dev/documentation/rxdart/latest/rx/BehaviorSubject-class.html)
/// in RxDart / ReactiveX.
///
```